### PR TITLE
make horizon url configurable

### PR DESCRIPTION
**Pull request by DylanVerstraete**

### CODE REVIEW

File: cmds/stellar/multisig.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func create(c *cli.Context) error {
 		return err
 	}
 
-	wallet, err := stellar.New(seed, network, nil)
+	wallet, err := stellar.New(seed, network, nil, "")
```
Comment (muhamadazmy) [inline cmds/stellar/multisig.go:34]: Why this is passed as empty string. I assume once the horizon url is provided in cmdline it must be used in all places. and not sometimes use it and others not

Comment (DylanVerstraete) [reply cmds/stellar/multisig.go:34]: this multisig command line tool is not used by anyone

File: pkg/escrow/stellar_test.go [hunk 2]
```diff
@@ -87,7 +87,7 @@ func TestPayoutDistribution(t *testing.T) {
 		assert.NoError(t, pd.Valid())
 	}
 
-	w, err := stellar.New("", stellar.NetworkTest, nil)
+	w, err := stellar.New("", stellar.NetworkTest, nil, "")
```
Comment (muhamadazmy) [inline pkg/escrow/stellar_test.go:90]: And here?

File: pkg/stellar/validation.go [hunk 3]
```diff
@@ -17,7 +17,7 @@ type AddressValidator struct {
 
 // NewAddressValidator creates an address validator instance
 func NewAddressValidator(network, assetCode string) (*AddressValidator, error) {
-	w, err := New("", network, nil)
+	w, err := New("", network, nil, "")
```
[the remark this answers is not in the document]
Comment (DylanVerstraete) [reply pkg/stellar/validation.go:21]: Now passing horizon url from main as well

**Review by LeeSmet — APPROVED** [review]

**Pull request merged**